Tensile/KernelWriter.py

```diff
@@ -458,7 +491,7 @@ def makeSubIterSchedule(self, kernel, localReadCode, iteration, pointerCode, wai
         for i in range(numMfmaPerIter):
           readLeft = readsPerIter
           # with PrefetchLocalRead, localreads can interleave with mfma
-          if kernel["PrefetchLocalRead"]:
+          if kernel["PrefetchLocalRead"]%kernel["LoopIters"]:
```

Comment:
kernel["PrefetchLocalRead"]%kernel["LoopIters"]
appears in a lot of places, might be worthwhile assigning it a new name in kernel parameters.

Reply:
done